src/resolvers/lens.ts

```diff
@@ -1,19 +1,33 @@
 import axios from 'axios';
-import { getAddress } from '@ethersproject/address';
+import { getAddress, isAddress } from '@ethersproject/address';
 import { getUrl, resize } from '../utils';
 import { max } from '../constants.json';
 
 const API_URL = 'https://api.lens.dev';
 
+function getDefaultImage(picture) {
+  if (picture?.original) return picture.original.url;
+  if (picture?.uri) return picture.uri;
+
+  return null;
+}
+
 export default async function resolve(address) {
+  const path = isAddress(address) ? 'defaultProfile' : 'profile';
+
+  const request =
+    path === 'defaultProfile'
+      ? `${path}(request: { ethereumAddress: "${getAddress(address)}"})`
```

Comment:
I've noticed that on Lenster UI https://lenster.xyz/u/fabien.lens the request is actually doing something like: 
```
profile(request: { ownedBy: "${address}"})
```
Using the Ethereum address, maybe this help getting avatar even if the image is not set as default